### ISSUE TITLE
pre-trained model of stage 1

### ISSUE THREAD

**Issue by Lynne-yyq** [RuizeHan/PAR#11]
Hello author,

Thank you for providing the source code. I have a few questions:

Is the pre-trained model required in the first stage? Which pre-trained model did you use?
Where should the pre-trained model be placed for use during the first stage?

Do you have a 'test_net'? Could you please provide it?
Thank you.

**Comment by RuizeHan** [comment]
We uploaded the base model of stage I to the cloud storage. Put this file into the path ./data.

https://pan.baidu.com/s/1eW9uj7wO8vaFgWSoRD-UeA @ PHAR.

**Comment by Lynne-yyq** [comment]
> We uploaded the base model of stage I to the cloud storage. Put this file into the path ./data.
> 
> https://pan.baidu.com/s/1eW9uj7wO8vaFgWSoRD-UeA @ PHAR.

Thank you very much for your reply. Are you referring to the first-stage model? My question is about whether a pre-trained model is needed during the training of the first stage.

**Comment by RuizeHan** [comment]
Yes, a pre-trained model (for appearance-based individual action feature extraction) is needed during the training of the first stage.
